tooling/Microsoft.VisualStudio.BlazorExtension/Properties/AssemblyInfo.cs

```diff
@@ -5,20 +5,6 @@
 
 // Add binding redirects for each assembly we ship in VS. This is required so that these assemblies show
 // up in the Load context, which means that we can use ServiceHub and other nice things.
-[assembly: ProvideBindingRedirection(
-    AssemblyName = "Microsoft.AspNetCore.Blazor.AngleSharp",
-    GenerateCodeBase = true,
-    PublicKeyToken = "",
-    OldVersionLowerBound = "0.0.0.0",
-    OldVersionUpperBound = "0.9.9.0",
-    NewVersion = "0.9.9.0")]
-[assembly: ProvideBindingRedirection(
-    AssemblyName = "Microsoft.AspNetCore.Blazor.Razor.Extensions",
-    GenerateCodeBase = true,
-    PublicKeyToken = "",
-    OldVersionLowerBound = "0.0.0.0",
-    OldVersionUpperBound = "1.0.0.0",
-    NewVersion = "1.0.0.0")]
```

Comment:
FYI @alexgav - this is how we register binding redirects for assemblies. The VSSDK adds these to the pkgdef that it generates and then eventually to the global VS `.exe.config`